apps/condo/domains/marketplace/access/InvoiceContext.js

```diff
@@ -17,7 +17,6 @@ const { RESIDENT } = require('@condo/domains/user/constants/common')
 async function canReadInvoiceContexts ({ authentication: { item: user } }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
-    if (user.type === RESIDENT) return false
```

Comment:
You need some special access if you want to give access to residents. And tests

Reply:
This checking returns nothing for the resident because the resident is not an employee with `canReadInvoiceContexts` permission. Similar checking we use while checking access to BillingIntegrationOrganizationContexts.